### ISSUE TITLE
Bedrockify 0.2.3 causes main menu panorama to stop.

### ISSUE THREAD

**Issue by K0-RR** [juancarloscp52/BedrockIfy#50]
The panorama is standing still instead of being animated.

**Comment by juancarloscp52** [comment]
Hi! Do you have other mods installed? Also, could you share with me a screenshot with your graphic settings?

**Comment by K0-RR** [comment]
By graphics settings do you mean bedrockify config? It should be default but yes I can send it if it’s needed.

About mods, yes I have more than 200 mods but (I will check this to be sure) I think previous bedrockify version was fine.

**Comment by juancarloscp52** [comment]
By Graphics settings I mean video settings, as with the last bedrockIfy version, panorama background depends more on how the graphic settings are set and the FPS of the client.

The mod list is needed in order to confirm that the issue is compatibility with other mods or if it is an issue with bedrockIfy it self.

Thanks for your time!

**Comment by JayCeeCreates** [comment]
I have the same issue as well.

Removed all the mods
Reset all configs
No resource packs loaded
The problem still persists. Panorama doesn't animate.

**Comment by juancarloscp52** [comment]
This issue is going to be hard to debug as I cannot reproduce this problem in any of my systems. Until I find the origin of the issue and a fix, you can disable the `client.features.panoramaBackground` mixin in the Mixins BedrockIfy Setting screen.

**Comment by juancarloscp52** [comment]
Okey, so I've been looking into this issue recently and I have been successfully force this issue, but I still need more information...

So, lets try something:
1. Go to your .minecraft/config folder and delete the file bedrockIfyMixin.json.
2. Start the game without any other mod installed.
3. Does the panorama background move? If not, please check in bedrockIfy settings > mixins and make sure that client.core.clientRenderTimer is enabled and restart.

Please tell me if this worked, with or without other mods, and if it doesn't please send a full log of starting the game until the title screen.

**Comment by K0-RR** [comment]
I just tried Bedrockify 1.1.0 on a fresh install on my laptop, with a few optimization mods and it's moving. (Possibility of being fixed in v.1.1.0?) When I was reporting this issue I was using my PC.
I will test this later today on the same computer as when I was reporting this issue. If it won't be moving I should delete the bedrockIfyMixin.json file and test again, right?

**Comment by juancarloscp52** [comment]
Yes! If it is not working, please include the game log and PC specifications (GPU, CPU, OS, RAM) if possible.

**Comment by K0-RR** [comment]
So this is weird, was testing this on my pc and the panorama was moving after disabling the Fabrication mod, there are 80 other mods running. Then I added fabrication on my laptop and panorama was still moving so I don't understand...

PC:
- CPU: Intel i7-6700K
- GPU: GTX 1070
- OS: Pop OS 20.10
- RAM: DDR4 HyperX 4x8GB - KHX2400C12D4

**Comment by K0-RR** [comment]
Removed all mods except Bedrockify, Fabrication and FAPI, still not moving on this pc. Should I do these "delete the file bedrockIfyMixin.json" steps?

log:

<details>

```
Using onesix launcher.

[19:36:52] [main/INFO]: Loading for game Minecraft 1.16.5
[19:36:52] [main/INFO]: [FabricLoader] Loading 53 mods: fabric-renderer-api-v1@0.4.1+ca58154a7d, fabric-keybindings-v0@0.2.1+ca58154a7d, fabricloader@0.11.3, fabric-structure-api-v1@1.1.4+ca58154a7d, fabric-containers-v0@0.1.11+9354966b7d, fabric-dimensions-v1@2.0.6+9354966b7d, fabric-object-builder-api-v1@1.9.4+9354966b7d, java@11, cotton-client-commands@1.0.1+1.16-rc1, fabric-game-rule-api-v1@1.0.6+ca58154a7d, fabric-api-base@0.2.1+9354966b7d, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, fabric-textures-v0@1.0.6+ca58154a7d, fabrication@1.3.4, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, fabric-renderer-registries-v1@2.2.1+ca58154a7d, fabric-loot-tables-v1@1.0.2+ca58154a7d, fabric@0.32.5+1.16, pride@1.0.0+1.16, fabric-particles-v1@0.2.4+ca58154a7d, fabric-networking-v0@0.3.2+ca58154a7d, fabric-api-lookup-api-v1@1.0.0+dc716ea17d, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, fabric-screen-handler-api-v1@1.1.5+9354966b7d, minecraft@1.16.5, cloth-basic-math@0.5.1, fabric-command-api-v1@1.1.1+351679a77d, fabric-screen-api-v1@1.0.0+c045166c7d, fabric-tool-attribute-api-v1@1.2.6+ca58154a7d, fabric-renderer-indigo@0.4.4+ca58154a7d, fabric-crash-report-info-v1@0.1.3+ca58154a7d, fabric-events-interaction-v0@0.4.2+ca58154a7d, fabric-item-api-v1@1.2.1+ca58154a7d, fabric-entity-events-v1@1.0.3+ca58154a7d, fabric-rendering-v0@1.1.2+ca58154a7d, fabric-key-binding-api-v1@1.0.4+9354966b7d, fabric-rendering-v1@1.5.1+ca58154a7d, cloth-config2@4.11.19, fabric-resource-loader-v0@0.4.2+ca58154a7d, fabric-content-registries-v0@0.2.1+ca58154a7d, fabric-tag-extensions-v0@1.1.1+ca58154a7d, fabric-biome-api-v1@3.1.1+ca58154a7d, fabric-commands-v0@0.2.2+ca58154a7d, fabric-registry-sync-v0@0.7.4+ca58154a7d, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-networking-api-v1@1.0.1+ca58154a7d, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, fabric-item-groups-v0@0.2.5+351679a77d, fabric-models-v0@0.2.1+ca58154a7d, fabric-object-builders-v0@0.7.2+ca58154a7d, bedrockify@1.1.0+mc1.16.2-5
[19:36:52] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/home/k0rr/.local/share/multimc/libraries/net/fabricmc/sponge-mixin/0.9.2+mixin.0.8.2/sponge-mixin-0.9.2+mixin.0.8.2.jar Service=Knot/Fabric Env=CLIENT
[19:36:52] [main/INFO]: [Fabrication] Reloading configs...
[19:36:52] [main/INFO]: [Fabrication] Loading of features.ini done in 7.515 ms with 0 warnings
[19:36:52] [main/WARN]: Overriding InjectionInfo for @Inject with com.unascribed.fabrication.support.injection.FailsoftCallbackInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo)
[19:36:52] [main/WARN]: Overriding InjectionInfo for @ModifyArg with com.unascribed.fabrication.support.injection.FailsoftModifyArgInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgInjectionInfo)
[19:36:52] [main/WARN]: Overriding InjectionInfo for @ModifyArgs with com.unascribed.fabrication.support.injection.FailsoftModifyArgsInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgsInjectionInfo)
[19:36:52] [main/WARN]: Overriding InjectionInfo for @Redirect with com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.RedirectInjectionInfo)
[19:36:52] [main/WARN]: Overriding InjectionInfo for @ModifyVariable with com.unascribed.fabrication.support.injection.FailsoftModifyVariableInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyVariableInjectionInfo)
[19:36:52] [main/WARN]: Overriding InjectionInfo for @ModifyConstant with com.unascribed.fabrication.support.injection.FailsoftModifyConstantInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo)
[19:36:52] [main/WARN]: Reference map 'pride-refmap.json' for pride.mixins.json could not be read. If this is a development environment you can ignore this message
[19:36:53] [main/WARN]: @Mixin target net/minecraft/class_3898$class_3208 is public in fabric-networking-api-v1.mixins.json:accessor.EntityTrackerAccessor and should be specified in value
[19:36:56] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[19:36:57] [main/INFO]: Setting user: _K0RR_
[19:36:57] [main/INFO]: [Fabrication] Loading of yeet_recipes.ini done in 273.0 μs with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of block_logo.png done in 208.1 ms with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of block_logo.ini done in 214.8 ms with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of dimensional_tools.ini done in 11.06 ms with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of gear_components.ini done in 6.138 ms with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of classic_block_drops.ini done in 1.645 ms with 0 warnings
[19:36:58] [main/INFO]: [Fabrication] Loading of item_despawn.ini done in 303.0 μs with 0 warnings
[19:36:58] [main/INFO]: Initializing BedrockIfy.
[19:36:58] [main/INFO]: [Indigo] Registering Indigo renderer!
[19:36:59] [main/INFO]: Initializing BedrockIfy Client.
[19:36:59] [main/INFO]: Backend library: LWJGL version 3.2.2 build 10
[19:37:01] [main/INFO]: Narrator library successfully loaded
[19:37:01] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (Fabric Renderer API (v1), Fabric Key Bindings (v0), Fabric Loader, Fabric Structure API (v1), Fabric Containers (v0), Fabric Object Builder API (v1), Fabric Game Rule API (v1), Fabric API Base, Fabric Rendering Data Attachment (v1), Fabric Textures (v0), Fabrication, Fabric Rendering Fluids (v1), Fabric BlockRenderLayer Registration (v1), Fabric Lifecycle Events (v1), Fabric Renderer Registries (v1), Fabric Loot Tables (v1), Fabric API, Pride Lib, Fabric Particles (v1), Fabric Networking (v0), Fabric API Lookup API (v1), Fabric Networking Block Entity (v0), Fabric Screen Handler API (v1), Fabric Command API (v1), Fabric Screen API (v1), Fabric Tool Attribute API (v1), Fabric Renderer - Indigo, Fabric Crash Report Info (v1), Fabric Events Interaction (v0), Fabric Item API (v1), Fabric Entity Events (v1), Fabric Rendering (v0), Fabric Key Binding API (v1), Fabric Rendering (v1), Cloth Config v4, Fabric Resource Loader (v0), Fabric Content Registries (v0), Fabric Tag Extensions (v0), Fabric Biome API (v1), Fabric Commands (v0), Fabric Registry Sync (v0), Fabric Mining Levels (v0), Fabric Networking API (v1), Fabric Events Lifecycle (v0), Fabric Item Groups (v0), Fabric Models (v0), Fabric Object Builders (v0), BedrockIfy)
[19:37:08] [main/INFO]: OpenAL initialized.
[19:37:08] [main/INFO]: Sound engine started
[19:37:08] [main/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/blocks.png-atlas
[19:37:08] [main/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[19:37:08] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[19:37:08] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[19:37:08] [main/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[19:37:08] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[19:37:08] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[19:37:09] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[19:37:09] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[19:37:09] [main/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[19:37:13] [main/INFO]: Stopping!
Process exited with code 0.
```
</details>

**Comment by juancarloscp52** [comment]
yes, try deleting bedrockIfyMixin.json.
If the problem persist, It might be a incompatibility with Fabrication Mod.

**Comment by K0-RR** [comment]
>Go to your .minecraft/config folder and delete the file bedrockIfyMixin.json.
Start the game without any other mod installed.
Does the panorama background move? If not, please check in bedrockIfy settings > mixins and make sure that client.core.clientRenderTimer is enabled and restart.

Done this and it didn't fix the issue, the mixin was enabled btw

**Comment by juancarloscp52** [comment]
Ok... I still don't understand why it is not working. I'll add the previous method of rendering as an alternative option in bedrockIfy settings.

**Comment by K0-RR** [comment]
@juancarloscp52 sooo I updated/added a few mods and now the panorama is moving.

**Comment by JayCeeCreates** [comment]
Get this: maybe it depends on the performance of the computer or something? 🤔 Results were pretty inconsistent-

**Comment by Hebgbs** [comment]
Bedrockify 1.1.0, Minecraft 1.16.5, panorama doesn't ever move. No panorama options available.

I mean, it's _non-critical_ so this should be considered low priority, just some features are missing that are purely aesthetic.

**Comment by juancarloscp52** [comment]
Please, if someone can try the last 1.17 version of bedrockIfy and check if it works now with the option "Use Old Panorama Rendering" under "Other settings..." subcategory.

**Comment by Hebgbs** [comment]
_Strangely_ the problem had resolved itself in 1.16 and old style panorama being disabled doesn't reproduce my issue. _I do not understand what I had done_ all I know is I have no problems at the moment.

**Comment by Hebgbs** [comment]
> _Strangely_ the problem had resolved itself in 1.16[...]

_I spoke too soon._ I thought it had something with LazyDFU so I binned that, but it did nothing to fix the stuck panorama in 1.16. Seems to be an intermittent problem — the **worst** kind.
